control-plane/api-gateway/common/translation.go

```diff
@@ -107,14 +107,23 @@ func (t ResourceTranslator) toAPIGatewayListener(gateway gwv1beta1.Gateway, list
 	return api.APIGatewayListener{
 		Name:     string(listener.Name),
 		Hostname: DerefStringOr(listener.Hostname, ""),
-		Port:     int(listener.Port),
+		Port:     ToContainerPort(listener.Port),
 		Protocol: listenerProtocolMap[strings.ToLower(string(listener.Protocol))],
 		TLS: api.APIGatewayTLSConfiguration{
 			Certificates: certificates,
 		},
 	}, true
 }
 
+func ToContainerPort(portNumber gwv1beta1.PortNumber) int {
+	// We don't want to use privileged ports, they require privileged access.
+	// Add 2000 to any port less than 1024. This is consistent with other gateways in the market.
+	if portNumber < 1024 {
+		return int(portNumber) + 2000
```

Comment:
Noting that we talked earlier today about changing this to a configurable value that defaults to `0` -- meaning that port mapping for privileged ports is disabled -- but can be changed to whatever the user prefers.